src/app/pages/office/components/eta-card/eta-card.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { IonicModule } from '@ionic/angular';
+
+import { ETACardComponent } from './eta-card.component';
+
+describe('EtaCardComponent', () => {
+  let component: ETACardComponent;
+  let fixture: ComponentFixture<ETACardComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ETACardComponent],
+      imports: [IonicModule.forRoot()],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(ETACardComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  }));
+
```

Comment:
```async``` is deprecated which is visible in WebStorm
<img width="603" alt="Screenshot 2022-05-03 at 09 29 35" src="https://user-images.githubusercontent.com/33056264/166424344-dc6f7240-30c5-40e9-b21d-1222df2c04b6.png">

We should now be using ```waitForAsync``` from the same import location.

Can we please also re-write this setup to be the standard DES way i.e.

```

  configureTestSuite(() => {
    TestBed.configureTestingModule({
      declarations: [ETACardComponent],
      imports: [IonicModule],
    });
  });

  beforeEach(waitForAsync(() => {
    fixture = TestBed.createComponent(ETACardComponent);
    component = fixture.componentInstance;
  }));
```